apps/src/music/MusicView.jsx

```diff
@@ -520,164 +547,125 @@ class UnconnectedMusicView extends React.Component {
     this.setState({feedbackClicked: true});
   };
 
-  render() {
-    // The tutorial has a width:height ratio of 16:9.
-    const aspectRatio = 16 / 9;
-
-    // Let's minimize the tutorial width at 320px.
-    const minAppWidth = 320;
-
-    // Let's maximize the tutorial width at 1280px.
-    const maxAppWidth = 1280;
```

Comment:
Should we still constrain the width for very wide displays?

Reply:
Looks like this code wasn't being used at all actually (I didn't see any references to `containerWidth` in any of the components). Do you think we should still try to constrain? I could imagine that if you have a very wide display, it would be nice to have all that block space

Reply:
Oh, in that case, we can fix later if it proves to be an issue!